webapp/src/scenes/app/components/SimpleDeploymentDialog.js

```diff
@@ -0,0 +1,188 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { withStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import CustomDialog from '../../../components/CustomDialog';
+import KeyValueList from './KeyValueList';
+
+const styles = (theme) => ({
+  invalidInput: {
+    color: '#bd0000',
+  },
+});
+
+const LOCAL_STORAGE_IMAGE_NAMES = 'imageNames';
+let localStoredImageNames =
+  localStorage.getItem(LOCAL_STORAGE_IMAGE_NAMES) || '[]';
+localStoredImageNames = JSON.parse(localStoredImageNames);
+
+const LOCAL_STORAGE_IMAGE_CONFIGS = 'imageConfigs';
+let localStoredImageConfigs =
+  localStorage.getItem(LOCAL_STORAGE_IMAGE_CONFIGS) || '{}';
+localStoredImageConfigs = JSON.parse(localStoredImageConfigs);
+
+const initialState = {
+  imageName: '',
+  keyValuePairs: [],
+  additionalDialogComponentInput: {},
+  isKeyInvalid: false,
+};
+
+class SimpleDeploymentDialog extends Component {
```

Comment:
If I understand this correctly, this is a direct copy of the KeyValueDialog just without the part that allows to set environment variables. This is quite some code duplication, maybe we could just add a flag prop to the KeyValueDialog component to disable the environment variable part. (Then it might make sense to rename it DeploymentDialog instead of KeyValueDialog).
Alternatively it might also make sense to allow the user to set Environment variables for the Workspace? This way you could make a configurable workspace (e.g. COLOR_SCHEME = black)